conf/base.config

```diff
@@ -22,29 +22,30 @@ process {
   // Process-specific resource requirements
   withLabel:process_low {
     cpus = { check_max( 2 * task.attempt, 'cpus' ) }
-    memory = { check_max( 16.GB * task.attempt, 'memory' ) }
-    time = { check_max( 4.h * task.attempt, 'time' ) }
+    memory = { check_max( 14.GB * task.attempt, 'memory' ) }
+    time = { check_max( 6.h * task.attempt, 'time' ) }
   }
   withLabel:process_medium {
     cpus = { check_max( 6 * task.attempt, 'cpus' ) }
-    memory = { check_max( 48.GB * task.attempt, 'memory' ) }
+    memory = { check_max( 42.GB * task.attempt, 'memory' ) }
     time = { check_max( 8.h * task.attempt, 'time' ) }
   }
   withLabel:process_high {
     cpus = { check_max( 12 * task.attempt, 'cpus' ) }
-    memory = { check_max( 96.GB * task.attempt, 'memory' ) }
+    memory = { check_max( 84.GB * task.attempt, 'memory' ) }
     time = { check_max( 16.h * task.attempt, 'time' ) }
   }
   withLabel:process_long {
-    time = { check_max( 8.h * task.attempt, 'time' ) }
+    time = { check_max( 20.h * task.attempt, 'time' ) }
   }
   withName:get_software_versions {
     cache = false
   }
+
 }
 
-// Defaults config params, may be overwritten by later configs
 params {
```

Comment:
Should we remove these params from `base.config` and put them in `nextflow.config` with all other default params?

Reply:
I think this is more of an issue for the template really, as these come from there.

Originally I put them here so that they were alongside the resource requirements above, so that people would see them when tinkering / investigating.

Reply:
Yes, these should probably be changed in the template first and then synced to the pipeline on the next release.

Reply:
https://github.com/nf-core/tools/pull/443

Reply:
Ive changed this now in https://github.com/nf-core/chipseq/pull/115